plugins/Jobs.cpp

```diff
@@ -841,3 +856,25 @@ bool BedrockPlugin_Jobs::_hasPendingChildJobs(SQLite& db, int64_t jobID) {
     return !result.empty();
 }
 
+bool BedrockPlugin_Jobs::_isValidSQLiteDateModifier(const string& modifier) {
+    list<string> parts = SParseList(SToUpper(modifier));
+    for (const string& part : parts) {
+        // Simple regexp validation
+        if (SREMatch("^(\\+|-)\\d{1,3} (YEAR|MONTH|DAY|HOUR|MINUTE|SECOND)S?$", part)) {
+            continue;
+        }
+
```

Comment:
Please remove this blank line.  Basically, if it's important enough to separate with a blank line, it should be important enough to provide a new block comment (eg, a comment explaining with the subsequent block of code does).  If it's not important enough to provide its own block comment, then it doesn't deserve a blank line to separate it out into its own block of code.